src/models/connection.ts

```diff
@@ -0,0 +1,18 @@
+import * as mongoose from 'mongoose';
+
+export default (db: string) => {
+  const connect = () => {
+    mongoose
+      .connect(db, { useNewUrlParser: true, useUnifiedTopology: true })
+      .then(() => {
```

Comment:
We should probably decide whether to use async/await or then, or not care. I would lean towards using async/await